templates/markdown/.partials/message.md

```diff
@@ -1,9 +1,4 @@
-<h3 class="message__header">
-  {{messageName}}
-  {{#if message.deprecated}}
-  <em>Deprecated</em>
-  {{/if}}
-</h3>
+### {{#if message.deprecated}} *Deprecated*{{/if}} {{messageName}}
```

Comment:
I'd use parenthesis or square brackets around the word "deprecated" to make it easier to understand it's not actually part of the name. I know it wasn't there before, just taking advantage that we're changing it.

Reply:
Sure, changed it.